### PR TITLE
Fix user_created_notification failing on empty emails

### PR DESCRIPTION
**Pull request by charn**
If a superuser with an empty exists. `user_created_notification` will fail with `AnymailInvalidAddress`.

### CODE REVIEW

**Review by voneiden — APPROVED** [review]
looks m'kay to me

File: events/signals.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def user_created_notification(sender, instance, created, **kwargs):
             for item in user_model.objects.filter(is_superuser=True).values_list(
                 "email"
             )
```
Comment (jorilindell) [inline events/signals.py:46]: This is not really related to this PR. But at some point there was problems when trying to send emails to admin users and some of them didn't have email address. I'm not really sure is that fixed in code level or was it fixed by adding dummy email for each superuser.
